File: python/leetcode/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums):
        i = 0
        while i < len(nums):
            if i > 0 and nums[i] == nums[i - 1]:
                i -= 1
                del nums[i]
            i+= 1
        return len(nums)

File: java/leetcode/remove-duplicates-from-sorted-array.java
public class Solution {
    static int removeDuplicates(int[] nums) {
        int c = 0;
        int l = nums.length;
        int i = 0;
        for (int j = 1; j < l; j++) {
            if (nums[i] != nums[j]) {
                nums[++i] = nums[j];
                c = i;
            }
        }
        return c + 1;
    }
}
